gatling/user-files/DockerFile/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM denvazh/gatling
+
+ENV NODE_VERSION=10.15.1
+ENV PATH=/opt/node-v$NODE_VERSION-linux-x64/bin:$PATH
+
+ADD https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION.tar.gz /tmp/
+
+RUN apk add build-base python python-dev py-pip linux-headers nodejs-npm\
```

Comment:
Do we need the `nodejs-npm` package if we're downloading and installing node from their website anyway?

Reply:
Ok, after we tested it: we indeed download and install node from their website, compiling the sources directly, but that still do not give us npm, so this package is relevant :+1: